### DISCUSSION TITLE
What are the main differences between this library and the Serialize component?

### DISCUSSION THREAD

**Discussion by reinierbutot** [CuyZ/Valinor#359] [Q&A] [answered]
I was wondering what the main differences between this library and the Serialize component are. Because it isn't clear to me right away. The website also isn't very convincing on why I should choose this library.

**Comment by romm** [comment, marked as answer]
Hi @reinierbutot, as you are not the first one to ask this, I've added a chapter in the documentation, see #362. Do the details explained there seem relevant to you?
